lexicons/tools/ozone/set/get.json

```diff
@@ -0,0 +1,50 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.get",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get a specific set and its values",
```

Comment:
for automod, it would be good to distinguish between HTTP 404 (set doesn't exist) from other errors (server error, something wrong with query). IIRC we had some history of doing 400 instead of 404.

this isn't feedback on this PR or the lexicon, just thinking outloud for the implementation half of this in ozone repo.

Reply:
There still isn't a 404 in the standard xrpc error types we have on hand, and using it is a little confusing because the client converts it into an `XRPCNotSupported` error.  As you know, I am a fan of getting this going!  But that is the source of the awkwardness at this point.